openvino_tensorflow/kernels/encapsulate_op.cc

```diff
@@ -349,6 +349,10 @@ void NGraphEncapsulateOp::Compute(OpKernelContext* ctx) {
       }
 
       ov::Any any = ng_element->get_rt_info()["index"];
+      if (any.empty()) {
```

Comment:
Adding a comment here would help other developers to understand the reason for it.

Reply:
Added the comments